history.txt

```diff
@@ -1,6 +1,42 @@
 OMERO version history
 =====================
 
+5.1.4 (September 2015)
+----------------------
+
+A bug-fix release covering all components. Improvements include:
+
+-  channel buttons fixed in OMERO.web
+-  improved UI experience when moving annotated data between groups in
+   OMERO.web
+-  major improvement in loading annotations in the righthand panel of
```

Comment:
Maybe "Improved performance" instead of "Major improvement" since there's no change in UI.